ddtrace/internal/utils/__init__.py

```diff
@@ -39,3 +40,31 @@ def get_argument_value(
             return args[pos]
         except IndexError:
             raise ArgumentError("%s (at position %d)" % (kw, pos))
+
+
+def set_argument_value(
+    args,  # type: Tuple[Any, ...]
+    kwargs,  # type: Dict[str, Any]
+    pos,  # type: int
+    kw,  # type: str
+    value,  # type: Any
+):
+    # type: (...) -> Tuple[Tuple, Dict]
+    """
+    This function parses the value of a target function argument that may have been
```

Comment:
```suggestion
    This function parses the value of a target function argument that may have been
```

this sentence is hard to read... Maybe just simplify to something like "Returns a new args, kwargs with the given value updated"?